bolt-design-system/bolt
# Convert bolt-link to a web component

## Jira

 - Convert Link to web component: http://vjira2:8080/browse/BDS-533

## Summary

Convert `bolt-link` to a web component.

## Details

We're converting `bolt-link` from a fully server-side rendered component to a custom element with Web Component support. This will allow for greater control over the presentation and behavior of the component even after it has shipped. We're still including the necessary semantic element(s), e.g. an anchor tag, inside of the custom element so that the component's basic functionality is intact even if JS is disabled.

This work was initially added to PR #953 on the `feature/convert-to-web-components` branch. That PR and branch will be closed, as it included work related to the card component.

## How to test

 - Go to https://boltdesignsystem-uurekzdsrc.now.sh/pattern-lab/?p=viewall-components-link and confirm links and all variations are displaying properly.
 - Review [Files Changed](https://github.com/bolt-design-system/bolt/pull/963/files) and give feedback, comments if necessary.
